src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py

```diff
@@ -0,0 +1,199 @@
+    except AttributeError:
+        location = get_one_of_subscription_locations()
+    result = get_vm_sizes(location)
+    return [r.name for r in result]
+
+
+class SupportedImages(Enum):  # pylint: disable=too-few-public-methods
+    ubuntu_tls = "UbuntuLTS"
+    ubuntu_dsvm = "UbuntuDSVM"
+
+
+with ParametersContext(command='batchai cluster create') as c:
+    c.register_alias('resource_group', options_list=('--resource-group', '-g'), arg_type=resource_group_name_type)
+    c.argument('location', options_list=('--location', '-l'), arg_type=location_type,
+               help='Location. You can configure the default location using `az configure --defaults '
+                    'location=<location>` or specify it in the cluster configuration file.')
+    c.register_alias('cluster_name', options_list=('--name', '-n'), help='Name of the cluster.')
+    c.argument('user_name', options_list=('--user-name', '-u'),
+               help='Name of the admin user to be created on every compute node.', arg_group='Admin Account')
+    c.argument('ssh_key', options_list=('--ssh-key', '-k'),
+               help='SSH public key value or path.', arg_group='Admin Account')
+    c.argument('password', options_list=('--password', '-p'),
+               help='Password.', arg_group='Admin Account')
+    c.argument('image', options_list=('--image', '-i'), arg_group='Nodes',
+               help='Operation system.', **enum_choice_list(SupportedImages))
+    c.argument('vm_size', options_list=('--vm-size', '-s'),
+               help='VM size (e.g. Standard_NC6 for 1 GPU node)', completer=get_vm_size_completion_list,
+               arg_group='Nodes')
+    c.argument('min_nodes', options_list=('--min',),
+               help='Min nodes count.', type=int, arg_group='Nodes')
+    c.argument('max_nodes', options_list=('--max',),
+               help='Max nodes count.', type=int, arg_group='Nodes')
+    c.argument('nfs_name', options_list=('--nfs',),
+               help='Name of a file server to mount. If you need to mount more than one file server, configure them in '
+                    'a configuration file and use --config option.',
+               arg_group='File Server Mount')
+    c.argument('nfs_resource_group', options_list=('--nfs-resource-group',),
+               help='Resource group in which file server is created. Can be omitted if the file server and the cluster '
+                    'belong to the same resource group',
+               arg_group='File Server Mount')
+    c.argument('nfs_mount_path', options_list=('--nfs-mount-path',),
+               help='Relative mount path for nfs. The nfs will be available at '
+                    '$AZ_LEARNING_MOUNT_ROOT/<relative_mount_path> folder.',
+               arg_group='File Server Mount')
+    c.argument('azure_file_share', options_list=('--afs-name',),
+               help='Name of the azure file share to mount. Please provide BATCHAI_AZURE_STORAGE_ACCOUNT and '
```

Comment:
I recommend renaming the environment var name to `AZURE_BATCHAI_STORAGE_ACCOUNT`.

Reply:
done